device/bgp/bgpv4.yaml

```diff
@@ -1,21 +1,39 @@
 components:
   schemas:
-    Device.Bgpv4:
+    Router.Bgpv4:
       x-status: under-review
       x-include: 
-      - './bgp.yaml#/components/schemas/Device.Bgp'
+      - './bgp.yaml#/components/schemas/Router.Bgp'
       description: >-
         Container for emulated BGPv4 peers and routes.
       type: object
-      required: [local_address, dut_address]
+      required: [dut_address]
       properties:
-        local_address:
-          description: >-
-            Local IPv4 address of the emulated router
-          type: string
-          format: ipv4
         dut_address:
           description: >-
             IPv4 address of the BGP peer for the session
           type: string
           format: ipv4
+          default: 0.0.0.0
+      
+    Router.Bgpv4Iface:
+      description: >-
+        Container for emulated BGPv4 peers and routes on a single Ipv4 interface.
+      type: object
+      required: [ipv4_name]
+      properties:
+        ipv4_name: 
+          description: >-
+            The unique name of ipv4 or loopback object 
```

Comment:
Ip or IP?  I saw a few places where we used both interchangebly

Reply:
Ideally I'd prefer IP where we are saying things like "IP address".  It can also be IPv4 or IPv6 if the version needs to be indicated.  But we have too many places where Ip is used and I didn't want to point it out right now.  I think we need to do a thorough review of the descriptions for these type of things and fix it.  We can do it after this PR goes in.